TaskWithCharAndWchar_t/TaskWithCharAndWchar_t.cpp

```diff
@@ -0,0 +1,29 @@
+#include <iostream>
+#include <stdlib.h>
+#include <string>
+
+int main()
+{
+	const char* str = "Hello, world!"; // it works correctly
```

Comment:
please declare the string like const char str[] =. The same for wchar_t down the code..